deploy/kubernetes/base/role-leases.yaml

```diff
@@ -0,0 +1,11 @@
+---
+# Source: aws-ebs-csi-driver/templates/role-leases.yaml
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: default
```

Comment:
Please add ` | sed -e "/namespace: /d" ` to the `generate-kustomize` line that makes this file, so that this namespace gets tossed

Reply:
Done.